src/status_im/contexts/wallet/collectible/tabs/about/view.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.contexts.wallet.collectible.tabs.about.view
+  (:require [quo.core :as quo]
+            [quo.theme]
+            [react-native.core :as rn]
+            [status-im.contexts.wallet.collectible.tabs.about.style :as style]
+            [status-im.contexts.wallet.temp :as temp]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(def ^:private link-card-space 28)
+
+(defn- view-internal
+  []
+  (let [window-width (rf/sub [:dimensions/window-width])
+        item-width   (- (/ window-width 2) link-card-space)]
+    [:<>
+     [rn/view {:style style/title}
+      [quo/text
+       {:size   :heading-2
+        :weight :semi-bold}
+       (:title temp/collectible-about)]]
+     [rn/view {:style style/description}
+      [quo/text
+       {:size :paragraph-2}
+       (:description temp/collectible-about)]]
```

Comment:
We already have the description of collectible stored in `[:wallet :last-collectible-details :collectible-data :description]`, so I think it is better to use it instead of temp data

Reply:
Done